src/dialogs/ADialog.ts

```diff
@@ -2,16 +2,33 @@ import {offset} from '../utils';
 
 abstract class ADialog {
 
-  protected static readonly visiblePopups: HTMLElement[] = [];
+  static readonly visiblePopups: HTMLElement[] = [];
+
+  static removePopup(popup: HTMLElement) {
+    const index = ADialog.visiblePopups.indexOf(popup);
+    if(index > -1) {
+      ADialog.visiblePopups.splice(index, 1);
+      popup.remove();
+    }
+    if(ADialog.visiblePopups.length === 0) {
+      document.removeEventListener('keyup', escKeyListener);
+    }
+  }
 
   private static removeAllPopups() {
     ADialog.visiblePopups.splice(0, ADialog.visiblePopups.length).forEach((d) => {
       d.remove();
     });
+    document.removeEventListener('keyup', escKeyListener);
   }
 
-  protected static registerPopup(popup: HTMLElement) {
-    ADialog.removeAllPopups();
+  protected static registerPopup(popup: HTMLElement, replace: boolean = false) {
```

Comment:
shouldn't be `replace = true` the default value, since it is the more secure behavior?